filament/backend/src/Platform.cpp

```diff
@@ -82,6 +83,15 @@ Platform::~Platform() noexcept = default;
 DefaultPlatform* DefaultPlatform::create(Backend* backend) noexcept {
     SYSTRACE_CALL();
     assert_invariant(backend);
+
+#if defined(ANDROID)
+    char scratch[PROP_VALUE_MAX + 1];
+    int length = __system_property_get("debug.filament.backend", scratch);
+    if (length > 0) {
+        *backend = Backend(atoi(scratch));
```

Comment:
You should check the result of `atoi()`

Reply:
You can't.
`On success, the function returns the converted integral number as an int value.
If the converted value would be out of the range of representable values by an int, it causes undefined behavior.`

I think this is good enough for a debug feature.